src/components/charts/list/DiscoverCharts.tsx

```diff
@@ -575,30 +598,56 @@ function ChartListHeader({
     )
 }
 
-export function EmptyChartGroup() {
+export function EmptyChartGroup({
+    title,
+    removeLearnMore = false,
+    image,
+    onClickViewChartButton,
+    buttonText,
+    subTitle,
+    styles,
+}: {
+    title?: string
+    removeLearnMore?: boolean
+    image?: any
+    onClickViewChartButton?: () => void
+    buttonText?: string
+    subTitle?: string
+    styles?: {}
+}) {
     const { url } = useRouteMatch()
     return (
-        <div className="bcn-0 flex left br-8 mt-20 ml-20 mr-20" style={{ gridColumn: '1 / span 4' }}>
-            <img src={empty} className="" style={{ width: '200px', margin: '20px 42px' }} />
+        <div className={`bcn-0 flex left br-8 mt-20 ml-20 mr-20`} style={{ gridColumn: '1 / span 4', ...styles }}>
+            <img src={image || empty} className="" style={{ width: '200px', margin: '20px 42px' }} />
             <div>
-                <div className="fs-16 fw-6">Chart group</div>
-                <div className="cn-7">Use chart groups to preconfigure and deploy frequently used charts together.</div>
-                <a
-                    href={DOCUMENTATION.CHART_DEPLOY}
-                    rel="noreferrer noopener"
-                    target="_blank"
-                    className="learn-more__href"
-                >
-                    {' '}
-                    Learn more about chart groups
-                </a>
-                <NavLink
-                    to={`${url}/group/create`}
-                    className="en-2 br-4 bw-1 mt-16 cursor flex no-decor"
-                    style={{ width: '100px' }}
-                >
-                    <div className="fw-6 cn-7 p-6">Create group</div>
-                </NavLink>
+                <div className="fs-16 fw-6">{title || `Chart group`}</div>
+                <div className="cn-7">
+                    {subTitle || `Use chart groups to preconfigure and deploy frequently used charts together.`}
+                </div>
+                {!removeLearnMore && (
+                    <a
+                        href={DOCUMENTATION.CHART_DEPLOY}
+                        rel="noreferrer noopener"
+                        target="_blank"
+                        className="learn-more__href"
+                    >
+                        {' '}
+                        Learn more about chart groups
+                    </a>
+                )}
+                {onClickViewChartButton ? (
```

Comment:
Add a function type check here 'type of onClickViewChartButton'

Reply:
done